freqtrade/exchange/__init__.py

```diff
@@ -265,7 +275,8 @@ def symbol_price_prec(self, pair, price: float):
             price = ceil(big_price) / pow(10, symbol_prec)
         return price
 
-    def buy(self, pair: str, ordertype: str, amount: float, rate: float) -> Dict:
+    def buy(self, pair: str, ordertype: str, amount: float,
+            rate: float, time_in_force='gtc') -> Dict:
```

Comment:
I'm not sure if adding a default value here does provide value if we always set it anyway (and this is only called once)...